src/app/frontend/common/components/labels/labels_controller.js

```diff
@@ -19,6 +19,12 @@
 const alwaysVisibleLabelsNumber = 5;
 
 /**
+ * Regular expression matching URLs.
+ * @type {RegExp}
+ */
+const urlRegexp = new RegExp('(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?', 'i');
```

Comment:
Updated URL regex to the most accurate one from https://mathiasbynens.be/demo/url-regex. It's on MIT license, I've left comment in the code.